src/main/kotlin/data/model/Chunk.kt

```diff
@@ -0,0 +1,12 @@
+package data.model
+
+data class Chunk(
+        var id: Int,
+        val start: Int,
+        val end: Int,
+        val sort: Int,
+        val srcFile: String,
```

Comment:
srcFile should probably be selectedFile

Reply:
@sarabiaj The `srcFile` here corresponds to the source audio. The database base Content table links the user's takes (from the Take table) to a particular Chunk.